docs/fides/docs/tutorial/google.md

```diff
@@ -0,0 +1,95 @@
+# Add Google Analytics
+
+To better understand the behavior of the app's users, add Google Analytics to the app and a Fidesctl System resource to annotate it.
+
+## Define the App's Google Analytics Identifier
+
+Open the `flaskr/__init__.py` file in your favorite editor, and define the `GOOGLE_ANALYTICS_ID` constant below line 7:
+
+```python
+GOOGLE_ANALYTICS_ID = "UA-xxxxxxxxx-y"
+```
+
+In the `create_app` function defined on line 11, include the Google Analytics ID value in the application's configuration by adding the following line below line 17:
+
+```python
+GOOGLE_ANALYTICS_ID=GOOGLE_ANALYTICS_ID,
+```
+
+## Add the Google Analytics Script
+
+Open the `flaskr/templates/base.html` file in your favorite editor, and include the following at the beginning of the `<head>` tag:
+
+```html
+{% if config['GOOGLE_ANALYTICS_ID'] %}
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{ config['GOOGLE_ANALYTICS_ID'] }}"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag("js", new Date());
+        gtag("config", "{{ config['GOOGLE_ANALYTICS_ID'] }}");
+    </script>
+{% endif %}
+```
+
+## Annotate a Fidesctl System Resource
+
+To ensure that the app's policies can account for the data collected by Google Analytics, define a new Fidesctl System resource by adding a `google_analytics_system.yml` file to the `fides_resources` directory. This file should contain the following configuration:
```

Comment:
We should give another quick description of why these annotations make sense for GA. I'd probably recommend asking the reader to think about their own version - link to the documentation for the field reference (so you see what fields they track) and explain what `data_use` might be appropriate (e.g. why `improve` instead of `provide`)

In this case - I chose two declarations because I believed there were effectively two uses:
* tracking & reporting (e.g. using pseudonymous behavioural data to analyze how people use the site)
* deriving location (e.g. using IP address to derive a geographic location)

These felt like two separate purposes. I would also say there are lots of other data uses for GA, but we're not leveraging any of the other features of the service in this project so this is sufficient